src/components/dashboard/FeedItems/FeedModalItem.js

```diff
@@ -106,6 +134,26 @@ const FeedModalItem = (props: FeedEventProps) => {
   )
 }
 
+const getFeedTextStyles = () => ({
+  message: {
+    paddingBottom: 0,
+    flexShrink: 0,
+  },
+})
+
+const FeedText = withStyles(getFeedTextStyles)(({ theme, styles, isSmallDevice, children }) => (
```

Comment:
this component is used to show some additional seller's info (email, website).
`FeedText` name is very common & general for such component

need to replace it to the name having more exact sense. e.g. `SellerInfo` or `SellerInfoText`